### PR TITLE
README update: augmented instructions from ConsenSys template

### PR DESCRIPTION
**Pull request by mapachurro**
I've built out this README to include instructions for less-technical users to submit suggestions, as well as more detailed instructions for running locally, and a process for dealing with linter dictionary issues.

# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [ ] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

-made these edits in the browser, actually

## Describe the change

I brought in instructions from https://raw.githubusercontent.com/ConsenSys/docs-template/main/README.md and wrote some additional content specific to the Linea environment.

## Issue fixed

No current issue exists

## Impacted parts

For content changes:

- [ x] Documentation content
- [ ] Documentation page organization

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,89 @@
...
+You can find our contribution guidelines [here](https://docs-template.consensys.net/category/getting-started) and our style guide [here](https://docs-template.consensys.net/getting-started/style-guide).
+
+Take a look at some [good first issues](https://github.com/ConsenSys/doc.zk-evm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to get started.
+
+### How to submit a suggestion or change
+The best way to suggest a change to these docs is through a process known as a **pull request**. If you're not familiar with how that works, check out [GitHub's guide here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+If that process is too involved for you, you can always open a thread on the [Community forum](https://community.linea.build/), or a ticket on the [Support page](https://support.linea.build/hc/en-us). 
+
+If you **are** familiar with making a Pull Request, we **highly recommend that you run a version of these docs locally, and preview your changes locally, before submitting them**. In fact, it's part of the PR process.
```
Comment (jlwllmr) [inline README.md:25]: I think we should make pull request lowercase for consistency with the rest of the page

**Review by jlwllmr — CHANGES_REQUESTED** [review]
Just need to adjust the inconsistent capitalisation then we're good!

File: README.md (hunk 1 above)
Comment (mapachurro) [reply README.md:25]: big brain

**Review by mapachurro — COMMENTED** [review]
Thank you!

**Review by jlwllmr — APPROVED** [review]

**Pull request merged**